docs/resources/server/advancements.md

```diff
@@ -0,0 +1,173 @@
+Advancements
+============
+
+Advancements are tasks that can be achieved by the player which may advance the progress of the game. Advancements can trigger based on any action the player may be directly involved in.
+
+Data-Driven Advancements
+------------------------
+
+All advancement implementations within vanilla are data driven via JSON. This means that a mod is not necessary to create a new advancement, only a [Data pack][datapack]. A full list on how to create and put these advancements within the mod's `resources` folder can be found on the [Minecraft Wiki][wiki].
```

Comment:
Capitalization of `Data pack` to `data pack`? Also, we could omit "folder" from "the mod's `resources` folder".